src/Windows/Vision/PartialVisionWindowEvents.cpp

```diff
@@ -115,11 +115,20 @@ void VisionWindow::onComboBoxSelectColorTeam(Gtk::ComboBoxText *combobox) {
         pattern[ObjectType::Team].singleColorType = mainColorList[row];
         pattern[ObjectType::Team].singleColorRange = range;
 
-        if(pattern[ObjectType::Opponent].singleColorType == mainColorList[row]){
+        pattern[ObjectType::Opponent].id = ObjectType ::Opponent;
```

Comment:
Poderia utilizar um HashMap ao invés de acessar o array com in enum. Mas isso não é urgente. Só anota :D

Reply:
Beleza

Reply:
Boa observação não acha @pedro282